### DISCUSSION TITLE
Split Chat windows

### DISCUSSION THREAD

**Discussion by jasonsan9364** [LiveHelperChat/livehelperchat#2105] [Q&A]
![image](https://github.com/LiveHelperChat/livehelperchat/assets/156453430/13631ae0-b50f-41da-a59f-dad9a793ffd8)

I need help with the above mentioned subject. It used to be a single window per chat. But now I see split windows, which is very complicated for me to manage. Is there a way where I can only see 1 chat per tab?

**Comment by remdex** [comment]
You can change that view in dashboard options

**Reply by jasonsan9364** [reply]
@remdex - It worked! Thanks for the help! much appreciated.
